packages/discovery-react-components/src/components/DocumentPreview/components/PdfHighlight/PdfHighlight.tsx

```diff
@@ -123,7 +123,9 @@ const Highlight: FC<{
               className,
               shape.className,
               active && `${base}__item--active`,
-              active && activeClassName
+              active && activeClassName,
+              shape.facetId && `category_${shape.facetId} mf-highlight`,
+              shape.facetId && active && `category_${shape.facetId} mf-active`
```

Comment:
What is the `mf` prefix signify? Also, how is this class different than `${base}__item--active`? Seems odd to have 2 different "active" classes.

Reply:
Seems like you should just be able to get by with `shape.facetId && `category_${shape.facetId}` and making use of existing classes.

Reply:
- `${base}__item--active` applies to the default color scheme where all enrichments have the same color. 
- `mf` stands for matches found. It the facet information is available, enrichments will be colored by category.

Reply:
Latest naming: `${base}--active`